hunting-buddy.lic

```diff
@@ -68,6 +73,11 @@ class HuntingBuddy
       # Execute scripts to run before the hunt
       execute_actions(before_actions)
 
+      if stop_on_boxes && !need_boxes?
```

Comment:
I'm not a fan of having the logic in two places, but it makes sense as written.

Reply:
yeah. we toyed with this for a while. One is checking within the context of the current combat-trainer instance. The other is checking globally. Thought about passing as combat-trainer argument, but it didnt really solve anything.